src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -9,11 +9,11 @@ internal static class EndpointParameterEmitter
     internal static string EmitSpecialParameterPreparation(this EndpointParameter endpointParameter)
     {
         return $"""
-                        var {endpointParameter.Name}_local = {endpointParameter.AssigningCode};
+                        var {endpointParameter.HandlerArgument} = {endpointParameter.AssigningCode};
```

Comment:
I was actually thinking the other way around. `EndpointParameter` should be strictly about processing syntax to set state on the parameter objects and `EndpointParameterEmitter` should handle constructing the blocks of code.

I'll add a commit to showcase what this might look like.